pubsub/gossipsub/README.md

```diff
@@ -336,7 +337,7 @@ any other message in the regular flow, for any topic. This can lead to
 message rate reduction whenever there is some correlated flow between
 topics, and can be significant for densely connected peers.
 
-For piggyback implementation details, consult the go implementation.
+For piggyback implementation details, consult the [Go implementation](https://github.com/libp2p/go-floodsub/search?q=piggyback&unscoped_q=piggyback).
```

Comment:
that's not a terribly helpful link.

Reply:
Would https://github.com/libp2p/go-floodsub/blob/b96cf6df635722ec55bf65cd23736228dd0f850d/gossipsub.go#L311-L323 be OK?

Reply:
Just link to the `gossipsub.go` file, because the coalescing and piggybacking happens in a few places and the reader should better read the whole thing.  it's not that big anyway.